lib/mix/tasks/firmware.ex

```diff
@@ -112,6 +112,8 @@ defmodule Mix.Tasks.Firmware do
           [Path.expand(overlay)]
       end
 
+    _ = prevent_overlay_overwrites!(project_rootfs_overlay)
```

Comment:
```suggestion
    prevent_overlay_overwrites!(project_rootfs_overlay)
```

Move the ignored return value to inside the function. I don't know what this would return.